ecl/eclscheduler/eclscheduler.cpp

```diff
@@ -223,10 +244,22 @@ int main(int argc, const char *argv[])
     try
     {
         initClientProcess(serverGroup, DCR_EclScheduler);
-        Owned <IStringIterator> targetClusters = getTargetClusters("EclSchedulerProcess", globals->queryProp("@name"));
-        if (!targetClusters->first())
-            throw MakeStringException(0, "No clusters found to schedule for");
+
         CIArrayOf<EclScheduler> schedulers;
+
+#ifdef _CONTAINERIZED
+        Owned<IPTreeIterator> queues = globals->getElements("queues");
+        ForEach(*queues)
+        {
+            IPTree &queue = queues->query();
+            const char *qname = queue.queryProp("@name");
+            DBGLOG("Start listening to queue %s", qname);
+            Owned<EclScheduler> scheduler = new EclScheduler(qname);
```

Comment:
This means every listed scheduler, listens to every queue.. 
Should there be an option, like there is for eclccserver's to specify 'listen' set of queues?

Reply:
That was the intention.  I don't think we should have more than one scheduler defined.

I think eclccserver was implemented that way because compiles for different targets can have vastly different memory requirements.  eclscheduler should use very little memory, and causes problems if there is more than one running at once.

Reply:
>and causes problems if there is more than one running at once.

should we prevent replicas from being configured to >1 for that reason?